dev/com.ibm.ws.beanvalidation.v20_fat/fat/src/com/ibm/ws/bval/v20/fat/BeanVal20_CDI_EJB_Test.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.bval.v20.fat;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.addDirectory;
+import static com.ibm.websphere.simplicity.ShrinkHelper.buildDefaultApp;
+import static com.ibm.websphere.simplicity.ShrinkHelper.buildJavaArchive;
+import static com.ibm.websphere.simplicity.ShrinkHelper.defaultDropinApp;
+import static com.ibm.websphere.simplicity.ShrinkHelper.exportToServer;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import bval.v20.cdi.web.BeanValCDIServlet;
+import bval.v20.customprovider.CustomProviderTestServlet;
+import bval.v20.multixml.web.BeanValidationTestServlet;
+import bval.v20.valueextractor.web.ValueExtractorServlet;
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+
+//TODO: Delete this class and re-enable these tests in BeanVal20Test once CDI 3.0 and EJBLite 4.0 are ready.
+@SkipForRepeat({ SkipForRepeat.EE9_FEATURES })
+@RunWith(FATRunner.class)
+public class BeanVal20_CDI_EJB_Test extends FATServletClient {
```

Comment:
I guess this is a good compromise. If you are interested though, the class-level support for `@SkipForRepeat` on test servlets would be nice

Reply:
I couldn't get the class-level support for the @SkipForRepeat to work since that required the applications to start, and these apps depend on CDI and EJB to start up.